ethstorage/es-node
# fix issue 171 -- eta are not correct (less than 0) after a restart from accidently shutdown

See the issue from the related issue: https://github.com/ethstorage/es-node/issues/171

This issue is not easy to repro, my investigative result is that the estimated time value has overflowed.

**Analysis**:
estime is calculated in the report func, so if the value is investigative value, that is only caused by overflow.
`estTime := elapsed / time.Duration(synced+emptyFilled) * time.Duration(blobsToSync+synced+emptyFilled+emptyToFill)`

As the `estTime` is int64 which max value is `9,223,372,036,854,775,807`.  and `elapsed` type is `time.Duration` (value is `seconds * 10 ^ 9`). and the  `blobsToSync+synced+emptyFilled+emptyToFill = 16,777,216`. 
So if the `(seconds download use) / (synced+emptyFilled) > 550`, it will overflow.

How to test:
1. run the es-node and wait for it to start syncing. 
2. check the process of the es-node by the port it uses. ( `lsof -i :9111` ) then kill the process (`kill -9 <processid>`)
3. re-run the es-node and check the estimated time remaining; the value should be larger than 0.
